code/controllers/configuration.dm

```diff
@@ -177,7 +177,6 @@
 	var/emag_recharge_rate = 0
 	var/emag_recharge_ticks = 0
 
-	var/map_voting = 0
```

Comment:
why are you purging this config?

Reply:
it causes issues with map voting sometimes if a vote is called and canceled